packages/svelte/src/internal/client/runtime.js

```diff
@@ -276,6 +277,11 @@ export function update_reaction(reaction) {
 
 	reaction.f |= EFFECT_IS_UPDATING;
 
+	if (reaction.ac !== null) {
+		reaction.ac?.abort(STALE_REACTION);
+		reaction.ac = null;
+	}
```

Comment:
```suggestion
	reaction.ac?.abort(STALE_REACTION);
	reaction.ac = null;
```

I don't think the check is buying us anything here right?

Reply:
it's saving us from writing `reaction.ac`, which is more expensive than reading it. though it doesn't need to be optional